.github/bin/install-zig.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+case "$(uname)" in
+  Darwin*)   os='macos'   ;;
+  Linux*)    os='linux'   ;;
+  Windows*)  os='windows' ;;
+  MINGW*)    os='windows' ;;
+  MSYS_NT-*) os='windows' ;;
+  *)         os='linux'   ;;
+esac
+
+case "${os}" in
+  windows*) ext='zip'    ;;
+  *)        ext='tar.xz' ;;
+esac
+
+version='0.8.1'
+url="https://ziglang.org/download/${version}/zig-${os}-x86_64-${version}.${ext}"
```

Comment:
```suggestion
arch="$(uname -m)"

version='0.8.1'
url="https://ziglang.org/download/${version}/zig-${os}-${arch}-${version}.${ext}"
```

Reply:
We should handle architecture -- the most recent macs are all `aarch64`, it's not really niche anymore.

Reply:
True...

The way I handle it locally is [asdf](https://asdf-vm.com) and [asdf-zig](https://github.com/cheetah/asdf-zig) :shrug: